### PR TITLE
James/demos

### PR DESCRIPTION
**Pull request by james-vendia**
First 3 demos, covering several industries and coverage/levels.

### CODE REVIEW

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+      submissionTime
+      transactionId
+      version
+    }
+    error
+  }
+}
+
+```
+1. Confirm the `LoanPerformance` item is included in the list of all items
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:212]: Not sure why this is being reset to **1.**.

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    1. Clicking on `View history` from the detailed view shows all version of the `Loan` over time
+1. View the `Loan Perforamnce` items
+    1. Repeat the same sub-steps from above
+    
+### Explore Data using GraphQL Explorer
+You can view data using the GraphQL Explorer view from either node in the Uni.
+
+1. Click `GraphQL Explorer` on either node
+1. List all `Loan` items
+    ```
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:114]:
```suggestion
    ```graphql
```

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+          originalLoanToValue
+          originalUnpaidPrincipalBalance
+          originationDate
+        }
+      }
+    }
+    ```
+
+1. List all `LoanPerformance` items
+    ```
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:133]:
```suggestion
    ```graphql
```

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+          loanDelinquencyStatus
+          loanIdentifier
+          monthlyReportingPeriod
+          servicerId
+        }
+      }
+    }
+    ```
+1. Filter `LoanPerformance` items by `currentInterestRate`
+    ```
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:151]:
```suggestion
    ```graphql
```

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+1. Click `Save`
+1. Wait for the indicator in the bottom-right to provide confirmation
+1. View the new `LoanPerformance` entry in the table
+
+### Add Data using GraphQL Explorer
+You can add data using the GraphQL Explorer view from either node in the Uni.
+
+1. Click `GraphQL Explorer` on either node
+1. Add a `LoanPerformance` item using the mutation below
+```
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:195]:
```suggestion
```graphql
```

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+      transactionId
+      version
+    }
+    error
+  }
+}
+
+```
+1. Confirm the `LoanPerformance` item is included in the list of all items
+    ```
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:213]:
```suggestion
    ```graphql
```

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 7]
```diff
@@ -0,0 +1,308 @@
...
+1. Modify the `currentUnpaidPrincipalBalance` value and click `Save`
+1. Once the indicator in the bottom-right confirms the save, click the `View history` button
+1. Review the changes between the two versions of this record
+
+### Update Data using GraphQL Explorer
+You can update data using the GraphQL Explorer view from either node in the Uni.
+
+1. Click `GraphQL Explorer` on either node
+1. List all `LoanPerformance` records
+    ```
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:249]:
```suggestion
    ```graphql
```

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 8]
```diff
@@ -0,0 +1,308 @@
...
+          servicerId
+        }
+      }
+    }
+    ```
+1. Copy the `_id` value of one of the records - this is the record your will modify in the next step
+1. Execute this mutation to modify the `unpaidPrincipalBalance` of the `LoanPerformance` record identified in the previous step.
+    1. __NOTE__: The `id` value in the mutation below should be modified to reflect the `id` value you copied in the previous step.
+    
+   ```
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:270]:
```suggestion
   ```graphql
```

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 9]
```diff
@@ -0,0 +1,308 @@
...
+          submissionTime
+          transactionId
+          version
+        }
+        error
+      }
+    }    
+    ```
+1. Confirm the update `LoanPerformance` record is reflected in the list of all items
+     ```
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:288]:
```suggestion
     ```graphql
```

File: demos/food-and-beverage/simple-supplier-distributor/README.md [hunk 10]
```diff
@@ -0,0 +1,281 @@
...
+    1. Columns allow sorting
+    1. Clicking on an `_id` value shows a detailed view of the `Product`
+    1. Clicking on `View history` from the detailed view shows all version of the `Product` over time
+    
+### Explore Data using GraphQL Explorer
+You can view data using the GraphQL Explorer view from either node in the Uni.
+
+1. Click `GraphQL Explorer` on either node
+1. List all `Product` items
+    ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/simple-supplier-distributor/README.md:112]:
```suggestion
    ```graphql
```

File: demos/food-and-beverage/simple-supplier-distributor/README.md [hunk 11]
```diff
@@ -0,0 +1,281 @@
...
+            sku
+            supplier
+            promotionalContent
+         }
+      }
+   }
+    ```
+
+1. Filter `Product` items by `supplier`
+    ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/simple-supplier-distributor/README.md:131]:
```suggestion
    ```graphql
```

File: demos/food-and-beverage/simple-supplier-distributor/README.md [hunk 12]
```diff
@@ -0,0 +1,281 @@
...
+1. Click `Save`
+1. Wait for the indicator in the bottom-right to provide confirmation
+1. View the new `Product` in the table
+
+### Add Data using GraphQL Explorer
+You can add data using the GraphQL Explorer view from either node in the Uni.
+
+1. Click `GraphQL Explorer` on either node
+1. Add a `Product` item using the mutation below
+   ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/simple-supplier-distributor/README.md:168]:
```suggestion
   ```graphql
```

File: demos/food-and-beverage/simple-supplier-distributor/README.md [hunk 13]
```diff
@@ -0,0 +1,281 @@
...
+         _owner
+         submissionTime
+         transactionId
+         version
+       }
+     }
+   }   
+   ```
+1. Confirm the `Product` item is included in the list of all items
+    ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/simple-supplier-distributor/README.md:185]:
```suggestion
    ```graphql
```

File: demos/food-and-beverage/simple-supplier-distributor/README.md [hunk 14]
```diff
@@ -0,0 +1,281 @@
...
+1. Modify the `price` value and click `Save`
+1. Once the indicator in the bottom-right confirms the save, click the `View history` button
+1. Review the changes between the two versions of this record
+
+### Update Data using GraphQL Explorer
+You can update data using the GraphQL Explorer view from either node in the Uni.
+
+1. Click `GraphQL Explorer` on either node
+1. List all `Product` records
+    ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/simple-supplier-distributor/README.md:221]:
```suggestion
    ```graphql
```

File: demos/food-and-beverage/simple-supplier-distributor/README.md [hunk 15]
```diff
@@ -0,0 +1,281 @@
...
+            promotionalContent
+         }
+      }
+   }
+    ```
+1. Copy the `_id` value of one of the records - this is the record your will modify in the next step
+1. Execute this mutation to modify the `price` of the `Product` record identified in the previous step.
+    1. __NOTE__: The `id` value in the mutation below should be modified to reflect the `id` value you copied in the previous step.
+    
+   ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/simple-supplier-distributor/README.md:242]:
```suggestion
   ```graphql
```

File: demos/food-and-beverage/simple-supplier-distributor/README.md [hunk 16]
```diff
@@ -0,0 +1,281 @@
...
+         submissionTime
+         transactionId
+         version
+       }
+     }
+   }
+   ```
+   
+1. Confirm the update `Product` record is reflected in the list of all items
+     ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/simple-supplier-distributor/README.md:261]:
```suggestion
     ```graphql
```

File: demos/food-and-beverage/supplier-distributor-with-control/README.md [hunk 17]
```diff
@@ -0,0 +1,330 @@
...
+
+1. Open the GraphQL Explorer of the __SupplierNode__
+1. Remove all contents in the GraphQL Editor
+1. Copy the contents of [products.gql](resources/products.gql) into the GraphQL Editor
+1. Click the `>` button to submit the request to the __SupplierNode__
+
+### Confirm Products as the Supplier
+You can confirm the existence of the newly added Products by executing this GraphQL query from the GraphQL Explorer of the __SupplierNode__
+
+```
```
Comment (brian-vendia) [inline demos/food-and-beverage/supplier-distributor-with-control/README.md:57]:
```suggestion
```graphql
```

File: demos/food-and-beverage/supplier-distributor-with-control/README.md [hunk 18]
```diff
@@ -0,0 +1,330 @@
...
+  }
+}
+```
+
+Notice that all fields listed are populated in the response.
+
+### Confirm Products as the Distributor
+You can also confirm the existence of the newly added Products by executing this GraphQL query from the GraphQL Explorer of the __DistributorNode__
+
+```
```
Comment (brian-vendia) [inline demos/food-and-beverage/supplier-distributor-with-control/README.md:93]:
```suggestion
```graphql
```

File: demos/food-and-beverage/supplier-distributor-with-control/README.md [hunk 19]
```diff
@@ -0,0 +1,330 @@
...
+
+1. Open the GraphQL Explorer of the __DistributorNode__
+1. Remove all contents in the GraphQL Editor
+1. Copy the contents of [purchase_orders.gql](resources/purchase_orders.gql) into the GraphQL Editor
+1. Click the `>` button to submit the request to the __DistributorNode__
+
+### Confirm Purchase Orders as the Distributor
+You can confirm the existence of the newly added Purchase Orders by executing this GraphQL query from the GraphQL Explorer of the __DistributorNode__
+
+```
```
Comment (brian-vendia) [inline demos/food-and-beverage/supplier-distributor-with-control/README.md:151]:
```suggestion
```graphql
```

File: demos/food-and-beverage/supplier-distributor-with-control/README.md [hunk 20]
```diff
@@ -0,0 +1,330 @@
...
+    }
+  }
+}
+
+```
+
+### Confirm Purchase Orders as the Supplier
+You can confirm the existence of the newly added Purchase Orders by executing this GraphQL query from the GraphQL Explorer of the __SupplierNode__
+
+```
```
Comment (brian-vendia) [inline demos/food-and-beverage/supplier-distributor-with-control/README.md:182]:
```suggestion
```graphql
```

File: demos/food-and-beverage/supplier-distributor-with-control/README.md [hunk 21]
```diff
@@ -0,0 +1,330 @@
...
+1. Open the GraphQL Explorer of the __SupplierNode__
+1. Remove all contents in the GraphQL Editor
+1. Copy the contents of [deliveries.gql](resources/deliveries.gql) into the GraphQL Editor
+   1. Modify the `purchaseOrderId` values in the mutation to reflect the `_id` values copied in the previous section
+1. Click the `>` button to submit the request to the __SupplierNode__
+
+### Confirm Delivery as the Supplier
+You can confirm the existence of the newly added Delivery by executing this GraphQL query from the GraphQL Explorer of the __SupplierNode__
+
+   ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/supplier-distributor-with-control/README.md:253]:
```suggestion
   ```graphql
```

File: demos/food-and-beverage/supplier-distributor-with-control/README.md [hunk 22]
```diff
@@ -0,0 +1,330 @@
...
+       }
+     }
+   }
+   
+   ```
+
+### Confirm Delivery as the Distributor
+You can confirm the existence of the newly added Delivery by executing this GraphQL query from the GraphQL Explorer of the __DistributorNode__
+
+  ```
```
Comment (brian-vendia) [inline demos/food-and-beverage/supplier-distributor-with-control/README.md:286]:
```suggestion
  ```graphql
```

**Review by brian-vendia — CHANGES_REQUESTED** [review]
Looks good. Just a few comments.

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (james-vendia) [reply demos/finserv-and-insurance/simple-loan-marketplace/README.md:220]: Markdown allows you to lead all numbered lists with a `1.`  It helps in case you need to adjust numbering and don't want to re-number all elements of the numbered list.  For example, check this out:

```
1. First
1. Second
1. Third
```

1. First
1. Second
1. Third

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md (hunk 2 above)
Comment (james-vendia) [reply demos/finserv-and-insurance/simple-loan-marketplace/README.md]: Alright, I'll do it.  My IDE gets weird when I include GQL but don't link to a GQL schema file.

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md (hunk 1 above)
Comment (james-vendia) [reply demos/finserv-and-insurance/simple-loan-marketplace/README.md:222]: See above

[the remark this answers is not in the document]
Comment (james-vendia) [reply demos/finserv-and-insurance/simple-loan-marketplace/README.md:222]: I believe this is an indentation issue with the code snippet.  Let me try again.

File: demos/finserv-and-insurance/simple-loan-marketplace/README.md [hunk 23]
```diff
@@ -0,0 +1,327 @@
...
+### Clone with HTTPS
+
+```bash
+git clone https://github.com/vendia/examples.git
+```
+
+</details>
+
+## Step 1 - Create a Multi-Party Uni
+To create a Uni, you can use either the [Share Web Application](https://share.vendia.net) or the [Share CLI]((https://vendia.net/docs/share/cli).
```
Comment (brian-vendia) [inline demos/finserv-and-insurance/simple-loan-marketplace/README.md:51]:
```suggestion
To create a Uni, you can use either the [Share Web Application](https://share.vendia.net) or the [Share CLI](https://vendia.net/docs/share/cli).
```

**Review by brian-vendia — APPROVED** [review]
One minor nit that will affect the generated link. Once you push the update feel free to merge.

🚢
